src/relay.rs

```diff
@@ -211,14 +214,27 @@ impl<T: DuplexTransport + 'static> Network<T> {
     /// * `token` - Address of the ERC20 token contract to use
     /// * `relay` - Address of the ERC20Relay contract to use
     /// * `confirmations` - Number of blocks to wait for confirmation
+    #[allow(clippy::too_many_arguments)]
```

Comment:
Try putting `#[cfg_attr(feature = "cargo-clippy", allow(too_many_arguments))]` at the top of `impl<T: DuplexTransport + 'static> Network<T>`

Reply:
Thanks, updated